CMakeLists.txt

```diff
@@ -129,6 +129,13 @@ if(CMAKE_C_COMPILER_ID MATCHES "Clang")
         -Wno-sign-conversion
         -Wno-error=c11-extensions
     )
+    # Some headers such as glib-2.0 use identifier names that start with '_' followed by a capital letter.
+    # The C standard reserves names beginning with an underscore and various other combinations. 
+    # ISO/IEC 9899:1999 (aka C99 standard) §7.1.3 Reserved identifiers
```

Comment:
Could you please change the '§' character to "Section" or "Sec."? We are trying to use only ASCII characters in the source code.

Reply:
Yep. I removed '§' character. 
I think it will be better.